app/src/renderer/components/wallet/PageSend.vue

```diff
@@ -2,59 +2,59 @@
 
-    tm-part(title='Transaction Details')
-      tm-form-group(:error='$v.fields.zoneId.$error'
+    tm-part(title="Transaction Details")
+      tm-form-group(:error="$v.fields.zoneId.$error"
         v-if="config.devMode"
-        field-id='send-zone-id' field-label='Zone ID')
+        field-id="send-zone-id" field-label="Zone ID")
         tm-field#send-zone-id(
           type="select"
           v-model="fields.zoneId"
           :options="zoneIds"
           placeholder="Select zone...")
-        tm-form-msg(name='Zone' type='required' v-if='!$v.fields.zoneId.required')
-      tm-form-group(:error='$v.fields.address.$error'
-        field-id='send-address' field-label='Send To')
+        tm-form-msg(name="Zone" type="required" v-if="!$v.fields.zoneId.required")
+      tm-form-group(:error="$v.fields.address.$error"
+        field-id="send-address" field-label="Send To")
         tm-field-group
           tm-field#send-address(
-            type='text'
-            v-model='fields.address'
-            placeholder='Address')
-        tm-form-msg(name='Address' type='required' v-if='!$v.fields.address.required')
-        tm-form-msg(name='Address' type='bech32' :body="bech32error" v-else-if='!$v.fields.address.bech32Validate')
+            type="text"
+            v-model="fields.address"
+            placeholder="Address")
+        tm-form-msg(name="Address" type="required" v-if="!$v.fields.address.required")
+        tm-form-msg(name="Address" type="bech32" :body="bech32error" v-else-if="!$v.fields.address.bech32Validate")
 
-      tm-form-group(:error='$v.fields.amount.$error'
-        field-id='send-amount' field-label='Amount')
+      tm-form-group(:error="$v.fields.amount.$error"
+        field-id="send-amount" field-label="Amount")
         tm-field-group
           tm-field#send-amount(
-            type='number'
+            type="number"
             :max="max"
-            :min='max ? 1 : 0'
-            v-model='fields.amount'
-            placeholder='Amount')
-        tm-form-msg(name='Amount' type='required' v-if='!$v.fields.amount.required')
-        tm-form-msg(name='Amount' type='between' :min='max ? 1 : 0' :max='max'
-          v-if='!$v.fields.amount.between')
+            :min="max ? 1 : 0"
+            v-model="fields.amount"
+            placeholder="Amount")
+        tm-form-msg(name="Amount" type="required" v-if="!$v.fields.amount.required")
+        tm-form-msg(name="Amount" type="between" :min="max ? 1 : 0" :max="max"
+          v-if="!$v.fields.amount.between")
 
-      p(v-if='mockedConnector')
+      p(v-if="mockedConnector")
         span Try sending to the address "
         strong(style="font-weight: bold") cosmos1p6zajjw6xged056andyhn62lm7axwzyspkzjq0
-        span ", it's a friendly bot which will send the money back to you!
-      br(v-if='mockedConnector')
+        span ", it"s a friendly bot which will send the money back to you!
```

Comment:
what's up with the quotes in this file? this one is a double quote?

Reply:
Fixed. It was bc my linter suggested to change them to double quotes to see the colors from the pug file lol